packages/fxa-auth-server/lib/routes/recovery-key.js

```diff
@@ -347,70 +349,12 @@ module.exports = (
         );
 
         return {
-          exists: exists.exists,
+          exists: !!recoveryKeyRecordWithHint,
```

Comment:
Will this return false if the user has a recovery key but no hint? `exists` should return true if the user has a key, whether or not they stored an optional hint.

Reply:
The variable is the record so it returns false when there's no recovery key.